ru/ru_barcode_targets.py

```diff
@@ -522,7 +538,7 @@ def run(parser, args):
     # Parse configuration TOML
     # TODO: num_channels is not configurable here, should be inferred from client
     conditions, reference, caller_kwargs = get_barcoded_run_info(
-        args.toml, validate=False
+        args.toml, validate=False, odd_even=args.odd_even
```

Comment:
So we don't need to pass this here, removing the changes in `utils.py` and instead pass `args.odd_even` to `simple_analysis`